### PR TITLE
Reenable org.openhab.core.automation.integration.tests

### PR DESCRIPTION
**Pull request by wborn**
Signed-off-by: Wouter Born <github@maindrain.net>

### CODE REVIEW

**Comment by openhab-5iver** [comment]
Awesome! I had gotten about half of them working, but then could not get the IDE to run any tests. Glad someone could get a working IDE together... I'll happily delete that branch!

**Comment by cweitkamp** [comment]
Conflicts, you know. ;)

File: itests/org.openhab.core.automation.integration.tests/src/main/java/org/openhab/core/automation/integration/test/AutomationIntegrationTest.java [hunk 1]
```diff
@@ -356,60 +353,6 @@ public void assertThatARuleWithNonExistingModuleTypeHandlerIsAddedToTheRuleRegis
         assertThat(ruleEngine.getStatusInfo(rule.getUID()).getStatus(), is(RuleStatus.UNINITIALIZED));
     }
 
-    @Test
```
Comment (cweitkamp) [inline itests/org.openhab.core.automation.integration.tests/src/main/java/org/openhab/core/automation/integration/test/AutomationIntegrationTest.java]: I am wondering why you removed these. Don't we need them anymore?

Comment (wborn) [reply itests/org.openhab.core.automation.integration.tests/src/main/java/org/openhab/core/automation/integration/test/AutomationIntegrationTest.java]: For some reason stopping/starting the bundles doesn't work with the bnd test runner so I remove them. We can also add `@Ignore` them for now and see if someone else comes up with a solution some day. :wink:

Comment (cweitkamp) [reply itests/org.openhab.core.automation.integration.tests/src/main/java/org/openhab/core/automation/integration/test/AutomationIntegrationTest.java]: Thanks for clarification. I think it's okay to remove them.

**Pull request merged**